plugin/trino-oracle/src/test/java/io/trino/plugin/oracle/AbstractTestOracleTypeMapping.java

```diff
@@ -91,6 +91,8 @@
     private final ZoneId kathmandu = ZoneId.of("Asia/Kathmandu");
     private final LocalDateTime timeGapInKathmandu = LocalDateTime.of(1986, 1, 1, 0, 13, 7);
 
+    protected TestingOracleServer oracleServer;
```

Comment:
> Having this in subclasses ensures that the tests don't mistakenly directly use oracleServer

I prefer to ensure this by **review**, because in the type mapping test with inheritance relationship (e.g. SQL Server, Oracle), all the test code is written to the parent class, we are **not** writing tests in the  subclass now.

I think this inheritance relationship is mainly to be able to **test different versions of connector** (such as the 6th, 7th, 8th commit of this [PR](https://github.com/trinodb/trino/pull/11148)).

So for now it is feasible to put `oracleServer` in the parent class.

Reply:
> I prefer to ensure this by review,

@tangjiangling "Ensuring by review" is basically hard. If external developers want to run the Oracle mapping tests with their custom Oracle image (≠ `TestingOracleServer`), the original structure is better than the new one.

Reply:
Got it, I'll fix it as @hashhar suggested earlier.

Reply:
Rebased to fix this.